doc/source/reference/random/index.rst

```diff
@@ -73,6 +73,15 @@ See `new-or-different` for more information
   rg.standard_normal()
   rg.bit_generator
 
+Something like the following code can be used to support both ``RandomState``
+and ``Generator``, with the understanding that the interfaces are slightly
+different
+
+.. code-block:: python
+
+  func = getattr(rg, 'integers', getattr(rg, 'randint', None))
```

Comment:
`getattr(rg, 'integers', None) or rg.randint` would be safer and faster

Reply:
I think Eric's last version is an improvement.

Reply:
adopted